website/src/components/blueprints_docs/BlueprintImportCard.jsx

```diff
@@ -39,7 +39,7 @@ function BlueprintImportCard({ category, id }) {
                 />
               </a>
               <br />
-              <small>(Home Assistant 2021.3.0 or higher)</small>
+              <small>(Home Assistant 2024.10.0 or higher)</small>
```

Comment:
```suggestion
              <small>(Home Assistant 2021.3.0 or higher)</small>
```

This is relative to the actual blueprint import feature. We should add an extra requirement for the Home Assistant Version